modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -930,9 +930,11 @@ namespace
                 {
                     // FIXME: ASSERT(DATA), ASSERT(FLUIDDATA)
                     auto &fd     = fg.metadata(out_data_node).get<FluidData>();
-                    fd.latency   = out_latency;
+                    // If fluid node is external, it will be bound to a real image without
+                    // fluid buffer allocation, so set its latency to 0 not to confuse later latency propagation
```

Comment:
And what if this data object is also read by its producing island? Then this data should be preserved here.

The case:
```
+-isl0--------------+
| OP -> Data1 -> OP |
+--------:----------+
         :
        +-isl1--------------+
        | OP -> Data2 -> OP :
        +-------------------+
```

Reply:
Ok, I got that it seems to work